low-level/sspl-ll.spec

```diff
@@ -63,17 +63,23 @@ SSPL_BASE=${RPM_BUILD_ROOT}/opt/seagate/%{product_family}/sspl
 mkdir -p $SSPL_BASE
 mkdir -p ${RPM_BUILD_ROOT}/etc/{systemd/system,dbus-1/system.d,polkit-1/rules.d,sspl-ll/templates/snmp}
 cp -afv files/etc ${RPM_BUILD_ROOT}/
-cp -afv files/opt/seagate/sspl/* $SSPL_BASE/
-
-# Rename setup directory to bin directory and remove .py extension of sspl_setup files.
-mv $SSPL_BASE/setup $SSPL_BASE/bin
-mv $SSPL_BASE/bin/sspl_setup.py $SSPL_BASE/bin/sspl_setup
+cp -afv files/opt/seagate/sspl/conf $SSPL_BASE/
+mkdir -p $SSPL_BASE/bin
 
 # Copy the service into /opt/seagate/%{product_family}/sspl where it will execute from
 cp -rp __init__.py $SSPL_BASE
 mkdir -p $SSPL_BASE/low-level
 cp -rp . $SSPL_BASE/low-level
 
+# Coping independent executable script inside sspl/low-level to easier access core code access.
+SSPL_SETUP=$SSPL_BASE/low-level/files/opt/seagate/sspl/setup
+cp -p $SSPL_SETUP/generate_resource_health_view/resource_health_view $SSPL_BASE/low-level/
+cp -p $SSPL_SETUP/generate_sspl_bundle/sspl_bundle_generate $SSPL_BASE/low-level/
+cp -p $SSPL_SETUP/manifest_support_bundle $SSPL_BASE/low-level/
+cp -p $SSPL_SETUP/sspl_setup.py $SSPL_BASE/low-level/sspl_setup
+cp -p $SSPL_SETUP/consuldump.py $SSPL_BASE/low-level/
+cp -rp $SSPL_SETUP/pacemaker $SSPL_BASE/bin/pacemaker
```

Comment:
what is this pacemaker in sspl?

Reply:
In the old code of sspl.spec file I have seen this code. So I kept this folder as it is in sspl/bin directory.

Reply:
ok its a folder (created for sspl resource agent) and seems it just has a README now. we can avoid this in R2 as sspl will controlled by pacemaker (HA) using common systemd resource agent.

Reply:
Done